include/Core/UI/ListTable/LogListTable.php

```diff
@@ -240,6 +243,25 @@ protected function column_cb( $item ) {
 		);
 	}
 
+	/**
+	 * Markup for Status column.
+	 *
+	 * @since 2.4.0
+	 *
+	 * @access protected
+	 *
+	 * @param object $item
+	 *
+	 * @return string
+	 */
+	protected function column_sent_status( $item ) {
+		if ( 1 === absint( $item->result ) ) {
```

Comment:
What would happen for email logs that were captured before we implemented this field?

Will they now show as failed? Can we not show anything in that case?

Reply:
@sudar,

> What would happen for email logs that were captured before we implemented this field?

The resulting output will be the same as the failed emails.

Will they now show as failed? Can we not show anything in that case?

Okay, implemented.

![image](https://user-images.githubusercontent.com/17510033/61289944-3e3dce80-a7e8-11e9-9763-c7530b79db04.png)